apps/finance/app/src/components/Transfers.js

```diff
@@ -199,12 +206,12 @@ class Transfers extends React.Component {
                         key={transfer.transactionHash}
                         token={tokenDetails[toChecksumAddress(transfer.token)]}
                         transaction={transfer}
-                        wideMode={width > 768 + 219}
+                        wideMode={width > breakpoints.medium + 219}
```

Comment:
Could we set a constant for `219`?

Reply:
I ended up removing this entirely, since we don’t want `IdentityBadge` to be in wide mode anymore.